sdk/core/Azure.Core/src/Shared/OperationInternalOfT.cs

```diff
@@ -279,6 +280,22 @@ protected override async ValueTask<Response> UpdateStatusAsync(bool async, Cance
             }
         }
 
+        public virtual string GetOperationId()
+        {
+            var id = _operation.GetOperationId();
+            if (string.IsNullOrEmpty(id))
+            {
+                var serializeOptions = new JsonSerializerOptions { Converters = { new StreamConverter() } };
```

Comment:
Would probably be better to define a ResponseConverter so that we have more control over what gets serialized. For instance, Response has both ContentStream and a Content properties, but we only need to serialize one of them. I'd suggest just using the Content property which is already BinaryData.